src/main/java/software/aws/neptune/gremlin/GremlinQueryExecutor.java

```diff
@@ -21,6 +21,7 @@
 import org.apache.tinkerpop.gremlin.driver.Cluster;
 import org.apache.tinkerpop.gremlin.driver.Result;
 import org.apache.tinkerpop.gremlin.driver.SigV4WebSocketChannelizer;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Can we remove this dependency? We weren't using Jetbrains prior to this.

We do have Lombok, I think you could use that instead?

Reply:
addressed thanks